docs/docs/pyproject.md

```diff
@@ -300,4 +300,4 @@ build-backend = "poetry.core.masonry.api"
 !!!note
 
     If your `pyproject.toml` file still references `poetry` directly as a build backend,
-    you should update it to reference `poetry_core` instead.
+    you should update it to reference `poetry-core` instead.
```

Comment:
Should this change also be made on line 291?

Reply:
According to PyPI, yes: https://pypi.org/project/poetry-core/ and https://pypi.org/project/poetry_core/

But, I assume this has to be confirmed by the team of Poetry.